db/schema.rb

```diff
@@ -232,12 +240,12 @@
     t.integer "void_date_value_check"
     t.integer "housingneeds_type"
     t.integer "housingneeds_other"
-    t.index ["created_by_id"], name: "index_lettings_logs_on_created_by_id"
-    t.index ["location_id"], name: "index_lettings_logs_on_location_id"
-    t.index ["managing_organisation_id"], name: "index_lettings_logs_on_managing_organisation_id"
-    t.index ["old_id"], name: "index_lettings_logs_on_old_id", unique: true
-    t.index ["owning_organisation_id"], name: "index_lettings_logs_on_owning_organisation_id"
-    t.index ["scheme_id"], name: "index_lettings_logs_on_scheme_id"
+    t.index ["created_by_id"], name: "index_case_logs_on_created_by_id"
```

Comment:
This looks wrong to me? Why are these index names changing?

Reply:
Agreed! Resolved in [3c4d864](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/894/commits/3c4d864902e7430555c0a52095edee9c3fb0e8af).